docs/sphinx/source/contributing/style_guide.rst

```diff
@@ -0,0 +1,107 @@
+.. _documentation-and-style-guide:
+
+Documentation and style guide
```

Comment:
I'd like to call this just "style guide" to be honest but then I feel like the "building the documentation" doesn't really fit in here. Suggestions? Is this fine as it is?

Reply:
Writing code and documentation
Mastering code and docs
Nailing code and docs
Bilingual on code and docs (jk)
Code and documentation